dao/src/main/java/greencity/entity/order/Bag.java

```diff
@@ -76,7 +87,11 @@ public class Bag {
     @JoinColumn
     private Employee editedBy;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
why Lazy ?

Reply:
to make it more convenient to delete entities, I added LAZY

Reply:
@ospodaryk 
Than just add cascadeType. And please change fetchType to the EAGER